app-box/app-box.html

```diff
@@ -140,7 +140,7 @@
           this.disabled = true;
           this._elementTop = this._getElementTop();
           this._elementHeight = this.offsetHeight;
-          this._scrollingRegionHeight = (this.scroller === window) ? window.innerHeight : this.scroller.offsetHeight;
+          this._scrollingRegionHeight = this._scrollTargetHeight;
```

Comment:
`_scrollTargetHeight` might trigger layout but `_scrollingRegionHeight` is a cached value, maybe it could be renamed as `_cachedScrollTargetHeight`